evaluate/evaluateresults.py

```diff
@@ -0,0 +1,185 @@
+import os
+import json
+from datetime import datetime, timedelta
+import dateutil.parser
+
+
+null_tags = ["false-positive", "none", "no-tag"]
+
+
+class TrackResult:
+    def __init__(self, track_record, clip_start, clip_end):
+        """ Creates track result from track stats entry. """
+        # ops... must have lost time zone at some point, so I put it back here...
```

Comment:
I don't know what this comment is about. I don't see timezones being referenced anywhere.

Reply:
Left it in from previous code by mistake.